CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 0.9.0 - 2024-09-03
+- ⚠️ **BREAKING** Add the `BUILDKITE_SPLITTER_RESULT_PATH` required environment variable. See [Migrating to 0.9.0](https://github.com/buildkite/test-splitter/tree/db4cab8cd6c82392553cd80481cf75e3888c2f4c?tab=readme-ov-file#migrating-to-090).
+- Experimental support for Jest by setting `BUILDKITE_SPLITTER_TEST_RUNNER` to `jest`.
+- Update the retry behavior to only retry failed tests.
+- Update split-by-example behavior to perform more work server-side.
+- Fix issue printing dry-run errors.
+- Fix issue with `BUILDKITE_STEP_ID` presence validation.
+
```

Comment:
I updated the validation logic of `BUILDKITE_PARALLEL_JOB` and `BUILDKITE_PARALLEL_JOB_COUNT` in this [pr](https://github.com/buildkite/test-splitter/pull/182) that also need to go into 0.9.0. Do we need to say something about it?

Reply:
```suggestion
- Fix issue with `BUILDKITE_PARALLEL_JOB_COUNT` and `BUILDKITE_PARALLEL_JOB` validation.

```
Something like this?

Reply:
Maybe something like "Improve logging output for configuration errors" is fine?